src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1569,6 +1570,34 @@ public function getContent(bool $asResource = false)
         return $this->content;
     }
 
+    /**
+     * Gets the request body decoded as array, typically from a JSON payload.
+     *
+     * @throws JsonException When the body cannot be decoded to an array
+     */
+    public function getContentArray(): array
```

Comment:
the http client uses `toArray` (also the `getContent`)
https://github.com/symfony/symfony/blob/8d450130e0422a8e056a0667dd730300f6be7a73/src/Symfony/Contracts/HttpClient/ResponseInterface.php#L72
could use same method name?

Reply:
Sure